### DISCUSSION TITLE
Looking For Help EdgeGPT not Bard

### DISCUSSION THREAD

**Discussion by darnells68** [FuseFairy/DiscordBot-BardGPT#4] [General]
Good day. I tried your code and it runs well for the most part. The AI image generation works perfect. However the chat with bing features is giving a bit of an issue it will work sometimes depending on the question it will either answer the question with no issues but most of the time it will have an error saying the following 

Error: InvalidRequest: Culture is not supported. (Parameter 'name')
English-United States is an invalid culture identifier.

This is the response I get back from discord. Do you have an suggestions on a possible fix for this.

**Comment by FuseFairy** [comment]
I'm not sure what the cause is, I'll update the edgegpt api later and maybe it can be fixed.

**Reply by darnells68** [reply]
Thanks let me know when you have the chance to review it.

**Reply by FuseFairy** [reply]
update finish
